tests/integration/base_deploy_test.go

```diff
@@ -80,30 +80,64 @@ func HandlePanics(r interface{}, o contracts.TestOperator, t func() *testing.T)
 	if r != nil {
 		logger.Infof("unexpected panic occurred during test %s, --> %v", t().Name(), r)
 		t().Fail()
-		o.TearDown()
+		o.TearDownRook()
```

Comment:
`o` seems like too un-descriptive. Maybe even helpful to call it `op`?

Reply:
done.